src/test/java/jewellery/inventory/integration/SaleCrudIntegrationTest.java

```diff
@@ -104,6 +152,135 @@ void setUp() {
     getResourcesInUserResponseDto(pearlRequest);
   }
 
+  @Test
+  void removeOrganizationSaleAfterReturnAllResourcesAndProductsFromSaleInOrganization() {
+    productRequestDto.setOwnerId(organizationSeller.getId());
+    ProductsInOrganizationResponseDto productResponse =
+        createProductInOrganization(productRequestDto);
+    SaleRequestDto saleRequestDto =
+        getSaleInOrganizationRequestDto(
+            organizationSeller, buyer, productResponse, resourcesInOrganizationResponseDto);
+
+    ResponseEntity<OrganizationSaleResponseDto> saleResponse =
+        createSaleInOrganization(saleRequestDto);
+
+    ResponseEntity<ResourceReturnResponseDto> resourceReturnResponse =
+        createReturnResourceResponse(
+            saleRequestDto, Objects.requireNonNull(saleResponse.getBody()));
```

Comment:
Remove empty lines in preparation to have clear separation:
preparation

execution

assertion